tests/Oryx.Integration.Tests/DotNetCoreEndToEndTests.cs

```diff
@@ -13,6 +13,7 @@
 
 namespace Microsoft.Oryx.Integration.Tests
 {
+    [Trait("Category", "DOTNETCORE")]
```

Comment:
Let's keep the case in line with the class names - like `DotNetCore` instead of `DOTNETCORE`